static/templates/settings/organization-settings-admin.handlebars

```diff
@@ -54,11 +55,22 @@
 
       <label for="id_realm_create_stream_by_admins_only" class="inline-block"
         id="id_realm_create_stream_by_admins_only_label"
-        title="{{t 'If checked, only administrators may create new streams.'}}">
+        title="{{t 'If checked, only administrators may create new streams.' }}">
         {{t "Only admins may create streams" }}
       </label>
     </div>
     <div class="input-group">
+      <input type="checkbox" class="inline-block" id="id_realm_email_changes_disabled"
+             name="realm_email_changes_disabled"
+        {{#if realm_email_changes_disabled}}checked="checked"{{/if}} />
+
+      <label for="id_realm_email_changes_disabled" class="inline-block"
+        id="id_realm_email_changes_disabled_label"
+        title="{{t 'Prevent users from changing their email address.' }}">
+        {{t "Email changes disabled" }}
```

Comment:
'Prevent users from changing their email address.' should be the body text, and probably the title should be `If checked, users will be unable to change their email addres`.